.github/ISSUE_TEMPLATE/new-documentation.yaml

```
@@ -0,0 +1,49 @@
+name: Request documentation for an upcoming product change
+description: Request new documentation, best practices, or considerations about an existing Astro or Astronomer Software feature.
+assignees:
+  - jwitz
+  - kwcanuck
+labels: ["kind/enhancement"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Want document a feature that you own? Skip this template and use the [new feature documentation template](https://github.com/astronomer/docs/blob/main/new-feature-template.md) to get started.
+  - type: dropdown
+    id: impact
+    attributes:
+      label: What kind of change is it? 
+      multiple: false
+      options:
+        - Major new feature
```

```suggestion
        - Major new feature
```

Will users know the difference between major or minor? Should this be clarified? Is this distinction necessary?